src/vs/workbench/parts/files/browser/views/explorerView.ts

```diff
@@ -261,6 +264,15 @@ export class ExplorerView extends CollapsibleView {
 			needsRefresh = this.filter.updateConfiguration();
 		}
 
+		// Check if sortOrder value changed
+		const configSortOrder = configuration && configuration.explorer && configuration.explorer.sortOrder;
+		if (this.sortOrder !== configSortOrder) {
+			this.sortOrder = configSortOrder;
+			const sorter = this.getSorter(this.sortOrder);
+			(<Tree>this.explorerViewer).setSorter(sorter);
```

Comment:
I mean, move it to the sorter class. But you still need that code to trigger a refresh of the explorer when the sorting config changed and imho it is fine that the explorerView triggers this and not the sorter.